test/memoize.js

```diff
@@ -5,15 +5,15 @@ describe('memoize', () => {
     it('should call a 0 arg function once', () => {
         let haveCalled = false;
 
-        const func = () => {
+        const func = (x) => {
             if (haveCalled) throw new Error('oops');
             haveCalled = true;
         };
 
         const resultFunc = memoize(func);
 
-        resultFunc();
-        resultFunc();
+        resultFunc(undefined);
```

Comment:
I thought this would make a bit more sense that we are _actually_ passing an argument in, it's just the same argument both times. Is this better or worse for clarity?

Reply:
I'm leaning towards worse.  It might help make the "zero argument case can be treated like a one-arg case" jump easier, but I think it may be a bit of a red-herring when you're initially reading the test case.  They might get distracted wondering what `x` is for.